pkg/plugins/policies/meshloadbalancingstrategy/api/v1alpha1/validator.go

```diff
@@ -157,6 +157,9 @@ func validateLoadBalancer(conf *LoadBalancer) validators.ValidationError {
 	case RoundRobinType:
 	case RandomType:
 	case LeastRequestType:
+		if conf.LeastRequest != nil {
+			verr.AddError("leastRequest", validateLeastRequest(conf.LeastRequest))
+		}
```

Comment:
You already check if it's nil inside `validateLeastRequest`
```suggestion
		verr.AddError("leastRequest", validateLeastRequest(conf.LeastRequest))
```